llvm/lib/Target/PowerPC/PPC.td

```diff
@@ -489,6 +497,18 @@ def : Processor<"620", G3Itineraries, [Directive620,
 def : Processor<"750", G4Itineraries, [Directive750,
                                        FeatureFRES, FeatureFRSQRTE,
                                        FeatureMFTB]>;
+def : Processor<"gekko", G4Itineraries, [Directive750,
+                                         FeatureFRES, FeatureFRSQRTE,
+                                         FeatureMFTB, FeaturePaired,
+                                         FeaturePairedExt, FeatureDCBZL]>;
+def : Processor<"750cl", G4Itineraries, [Directive750,
+                                         FeatureFRES, FeatureFRSQRTE,
+                                         FeatureMFTB, FeaturePaired,
+                                         FeatureDCBZL]>;
+def : Processor<"broadway", G4Itineraries, [Directive750,
+                                            FeatureFRES, FeatureFRSQRTE,
+                                            FeatureMFTB, FeaturePaired,
+                                            FeaturePairedExt, FeatureDCBZL]>;
```

Comment:
I wonder if there any ISA differences between broadway and espresso, should check that

Reply:
doesn't look to be the case